packages/app/components/Queue/TA/TAQueueDetailButtons.tsx

```diff
@@ -84,6 +85,20 @@ export default function TAQueueDetailButtons({
     changeStatus(OpenQuestionStatus.Helping);
     openTeams();
   };
+  const deleteQuestion = async () => {
+    await changeStatus(
+      question.status === OpenQuestionStatus.Drafting
+        ? ClosedQuestionStatus.DeletedDraft
+        : LimboQuestionStatus.TADeleted
+    );
+    await API.questions.notify(question.id);
+  };
+
+  useHotkeys("shift+d", () => {
+    if (isCheckedIn) {
+      deleteQuestion();
+    }
```

Comment:
how does the library deal with asynchronous callbacks again? i forget exactly

Reply:
the `useHotkeys` callback is by default not async, but i think the contents inside the callback can be async and still work normally. [i think(?) i dk react LOL](https://dev.to/silvestricodes/asynchronous-flows-with-react-hooks-1g0m)